### DISCUSSION TITLE
Preview images inside kohya look way differently than images generate inside a1111, using the same checkpoint? What I'm doing wrong?

### DISCUSSION THREAD

**Discussion by dilectiogames** [bmaltais/kohya_ss#2879] [Q&A]
I'm trying to train embeddings for SD1.5,

Here is how a test image automatically generated by kohya looks, (I've added the censorship so I can upload it safely)

![a01](https://github.com/user-attachments/assets/88a828b7-de4f-4b68-bd8e-b51c1d1bd291)

and here how using the same prompt ( a single word the embedding name same one used in kohya Sample Prompts box) the same checkpoint, the same sampler inside automatic

![a02](https://github.com/user-attachments/assets/427bd64b-d5ac-4cc1-aece-d4b0996ccd67)

what prompts kohya is using to get those images, there are hidden prompts? I really doubt this but I really don't understand the difference.

Second thing is that the embedding is really small in file size is 1.614 bytes, looks to me there may be a mistake when saving the embeding? Do you know where i can increase this size? Maybe I've forgot to increase it and it doesn't have enough data?

Did kohya internal generate uses the file saved on this or something cached somewhere?

**Comment by nikolatesla20** [comment]
I see similar, the images from my sample prompts don't seem to look anything like what I'm training its really weird

**Comment by bru-ber** [comment]
This is quite a huge difference! What I can say is that prompts are even understood differently between A1111 and comfyui. One of the main reasons (that no one talks about) that people are finding it difficult to change from A1111 to comfy is that they need to adapt their prompting. I have nodes in comfy that let me switch between different prompt interpreters (A1111, Comfy, Comfy++,...) and each gives a very different image result.

**Reply by dilectiogames** [reply]
it was automatic1111, but i think is a thing from kohya, I need to install a more recent version maybe it got fixed
